src/components/Header/Navbar.tsx

```diff
@@ -10,11 +10,14 @@ import { useNavigate } from 'react-router-dom';
 import HomeIcon from '@mui/icons-material/Home';
 import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
 import Badge from '@mui/material/Badge/Badge';
-import TemporaryDrawer from '../Categories/CategoriesMenu';
+import BasicMenu from '../Categories/ListDividers';
```

Comment:
Don't change the naming in import - there component you import is called ListDividers